certifi/core.py

```diff
@@ -17,5 +17,4 @@ def where():
 
 
 def what():
-    with importlib.resources.open_binary("certifi", "cacert.pem") as fh:
-        return fh.read()
+    return importlib.resources.read_text("certifi", "cacert.pem")
```

Comment:
I don't think we want `utf-8` encoding but instead `ascii` because if unicode is in the PEM-encoded certificate, we want this to fail